bindings/python/test/coordinate/frame/providers/iers/test_manager.py

```diff
@@ -22,11 +22,22 @@
 
 
 @pytest.fixture
-def manager() -> Manager:
+def manager(bulletin_a) -> Manager:
     manager = Manager.get()
 
     manager.set_mode(Manager.Mode.Automatic)
-    manager.set_remote_url(URL.parse("https://maia.usno.navy.mil/ser7/"))
+    manager.set_bulletin_a_remote_url(
+        URL.parse(
+            "https://media.githubusercontent.com/media/open-space-collective/open-space-toolkit-data/main/data/coordinate/frame/providers/iers/bulletin-A/"
+        )
+    )
+    manager.set_finals_2000_a_remote_url(
+        URL.parse(
+            "https://media.githubusercontent.com/media/open-space-collective/open-space-toolkit-data/main/data/coordinate/frame/providers/iers/finals-2000A/"
+        )
+    )
+
+    manager.load_bulletin_a(bulletin_a)
```

Comment:
Added this line so that the python tests here do not have to fetch external data (the cpp tests do). A couple tests results changed due to this